modules/cnv-viewing-vmi-ip-web.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc
+
+[id="cnv-viewing-vmi-ip-web_{context}"]
+= Viewing the IP address of a virtual machine interface in the web console
+
+The IP information displays in the *Virtual Machine Details* screen for the virtual machine. 
+
+.Procedure
+
+. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
+. Click the virtual machine name to open the *Virtual Machine Details* screen.
+
+The information for each attached vNIC is displayed under *IP ADDRESSES*.
```

Comment:
Do we need an example output here, maybe with call outs to describe what is shown?

Reply:
It's just the IP addresses against the vNIC name displayed in the web console with other VM details.